server/views/applications/delete_key.njk

```diff
@@ -18,7 +18,10 @@
       <dt class="govuk-summary-list__key">
         Key type
       </dt>
-      <dd class="govuk-summary-list__value"> {{ this_data.keyType }}
+      <dd class="govuk-summary-list__value">
+        {{ 'Rest API key' if this_data.keyKind === 'api-client#key' }}
+        {{ 'Stream API Key' if key.kind === 'api-client#stream-key' }}
+        {{ 'Web API Key' if key.kind === 'api-client#web' }}
```

Comment:
Couldn't we have done the same mapping using the Key Type?

Reply:
I wanted to keep the same logic as on the view page so that behaviour was guaranteed to be the same. Which was Ieuans requirement.
If I knew if I could I'd have used a shared static function.

I forget if there was a specific reason I used kind rather than type in the view page.

Reply:
Shouldn't getting the key kind for the other two keys be the same as rest:
```suggestion
        {{ 'Web API Key' if this_data.keyKind === 'api-client#web' }}
```

Reply:
Good spot. Theres egg on my face.